triangle.js

```diff
@@ -0,0 +1,20 @@
+
+var isTriangle = function(a, b, c) {
+  if ((a <= 0) || (b <= 0) || (c <= 0)) {
```

Comment:
there's a lot of conditionals here - any way we could DRY this up? or make it easier to read possibly? could we be more explicit with our variable names?